dev-python/bokeh/files/bokeh-0.7.1-setup.patch

```diff
@@ -0,0 +1,10 @@
+--- setup.py	2015-01-12 19:37:38.000000000 +0100
```

Comment:
Please write a header, why this is needed, also see <http://dev.gentoo.org/~vapier/clean-patches>!